light-client/src/components/io.rs

```diff
@@ -1,20 +1,11 @@
 //! Provides an interface and a default implementation of the `Io` component
 
-use flex_error::{define_error, TraceError};
 use std::time::Duration;
 
-#[cfg(feature = "rpc-client")]
-use tendermint_rpc::Client;
-
-use tendermint_rpc as rpc;
+use flex_error::{define_error, TraceError};
 
 use crate::types::{Height, LightBlock};
-
-#[cfg(feature = "tokio")]
-type TimeoutError = flex_error::DisplayOnly<tokio::time::error::Elapsed>;
-
-#[cfg(not(feature = "tokio"))]
-type TimeoutError = flex_error::NoSource;
+use crate::utils::time::{timeout, TimeError};
```

Comment:
Can we feature-guard this import so that it doesn't trigger an unused warning when building for wasm?